### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by SeuRonao**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `main` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `main` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `main`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://docs.github.com/education/manage-coursework-with-github-classroom/leave-feedback-with-pull-requests)”.
</details>

Subscribed: @m-vinicius

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -6,6 +6,27 @@ def main():
...
+
+    quant_litros_galao = a / 6
+    galao = quant_litros_galao / 3.6
+    galao = math.ceil(galao)
+    preco_galao = galao * 25
+
+    print(f'O valor gasto comprando apenas latas é de R$ {preco_galao:.2f}.')
+    print(f'Serão necessárias {galao} latas.')
+
+    #O valor gasto comprando de forma que gere a menor quantidade de desperdício é de R$ [valor com duas casas decimais].Serão necessárias [quantidade] latas e [quantidade] galões.
```
Comment (SeuRonao) [inline main.py:28]: Claro que posso dar dica.
Para evitar desperdício você pode fazer assim.
Calcule quanto de latas precisa para cobrir toda a área deixando uma folga (`latas = litros_necessários // 18`) ele pega só a parte inteira da divisão, ou seja pode ter resto.
O resto que falta é justamente a quantidade de litros que tu vai ter que comprar em galões (arredondando pra cima, claro).

Outra coisa, olhando o seu código eu não vi aonde você coloca a folga de 10%.

Comment (m-vinicius) [reply main.py:28]: Certo professor, esqueci de colocar a folga de 10%.

File: main.py [hunk 2]
```diff
@@ -6,6 +6,37 @@ def main():
...
+
+    litros = ((quant_litros_latas * 1.1) // 18)
+    resto = (litros % 18)
+    litros = math.ceil(litros)
+    resto = math.ceil(resto)
+    valor_final = (litros * 80) + ((resto) * 25)
+
+    print(
+        f'O valor gasto comprando de forma que gere a menor quantidade de desperdício é de R$ {valor_final:.2f}. Serão necessárias {litros} latas e {resto} galões.'
```
Comment (SeuRonao) [inline main.py:37]: Aqui faltou um enter também.

File: main.py [hunk 3]
```diff
@@ -6,6 +6,36 @@ def main():
     """Função principal que será rodada quando o script for passado para o interpretador."""
     # COLOQUE SEU CÓDIGO AQUI
 
+    print('Área a ser pintada em metros quadrados:')
+    a = float(input('> '))
```
Comment (SeuRonao) [inline main.py:10]: A folga deveria e poderia ter sido dada logo na área.

File: main.py [hunk 4]
```diff
@@ -6,6 +6,36 @@ def main():
...
+    print(f'O valor gasto comprando apenas galões é de R$ {preco_galao:.2f}.')
+    print(f'Serão necessários {galao} galões.')
+
+    #O valor gasto comprando de forma que gere a menor quantidade de desperdício é de R$ [valor com duas casas decimais].Serão necessárias [quantidade] latas e [quantidade] galões.
+
+    litros = ((quant_litros_latas * 1.1) // 18)
+    resto = (litros % 18)
+    litros = math.ceil(litros)
+    resto = math.ceil(resto)
+    valor_final = (litros * 80) + ((resto) * 25)
```
Comment (SeuRonao) [inline main.py:34]: Honestamente, eu não entendi o que era a intenção aqui.
Poderia ter mudado isso para:

```suggestion
    latas = (a * 1.1 / 6) // 18
    resto = (a * 1.1 /6) % 18
    galoes = math.ceil(resto / 3.6)
    valor_final = latas * 80 + galoes * 25
```

Que eu acredito que funcionaria.

**Comment by SeuRonao** [comment]
Deadline atingido sem remoção dos erros finais.

**Pull request closed without merging**